recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.10.1" %}
+{% set version = "0.11.0" %}
 
 package:
   name: iris-grib
```

Comment:
Doesn't this version of iris-grib require iris ``2.*``?

Reply:
Erm... yes.
Will address ...